usaspending_api/api_contracts/contracts/v2/download/accounts.md

```diff
@@ -24,21 +24,28 @@ Generate files and return metadata using filters on custom account
 
```

Comment:
Do the values need to remain on lines 14 and 19 for dredd to pass? Perhaps including a body subsection would be better

Reply:
ignore